src/features/classification/security-controls/utils.js

```diff
@@ -15,47 +15,52 @@ import {
     SHARED_LINK_ACCESS_LEVEL,
 } from '../constants';
 
-const { SHARED_LINK, DOWNLOAD, EXTERNAL_COLLAB, APP } = ACCESS_POLICY_RESTRICTION;
+const { SHARED_LINK, DOWNLOAD, EXTERNAL_COLLAB, APP, WATERMARK } = ACCESS_POLICY_RESTRICTION;
 const { DEFAULT, WITH_APP_LIST, WITH_OVERFLOWN_APP_LIST } = APP_RESTRICTION_MESSAGE_TYPE;
 const { DESKTOP, MOBILE, WEB } = DOWNLOAD_CONTROL;
 const { BLOCK, WHITELIST, BLACKLIST } = LIST_ACCESS_LEVEL;
 const { COLLAB_ONLY, COLLAB_AND_COMPANY_ONLY, PUBLIC } = SHARED_LINK_ACCESS_LEVEL;
 
-const getShortSecurityControlsMessage = (controls: Controls): ?MessageItem => {
-    const { sharedLink, download, externalCollab, app } = controls;
+const getShortSecurityControlsMessage = (controls: Controls): Array<MessageItem> => {
+    const items = [];
+    const { sharedLink, download, externalCollab, app, watermark } = controls;
+
     // Shared link and external collab restrictions are grouped
     // together as generic "sharing" restrictions
     const sharing = (sharedLink && sharedLink.accessLevel !== PUBLIC) || externalCollab;
 
-    if (sharing && download && app) {
-        return { message: messages.shortAllRestrictions };
-    }
-
-    if (sharing && download) {
-        return { message: messages.shortSharingDownload };
-    }
-
-    if (sharing && app) {
-        return { message: messages.shortSharingApp };
-    }
-
-    if (download && app) {
-        return { message: messages.shortDownloadApp };
-    }
-
-    if (sharing) {
-        return { message: messages.shortSharing };
-    }
-
-    if (download) {
-        return { message: messages.shortDownload };
+    switch (true) {
+        case !!(sharing && download && app):
+            items.push({ message: messages.shortAllRestrictions });
+            break;
+        case !!(sharing && download):
+            items.push({ message: messages.shortSharingDownload });
+            break;
+        case !!(sharing && app):
+            items.push({ message: messages.shortSharingApp });
+            break;
+        case !!(download && app):
+            items.push({ message: messages.shortDownloadApp });
+            break;
+        case !!sharing:
+            items.push({ message: messages.shortSharing });
+            break;
+        case !!download:
+            items.push({ message: messages.shortDownload });
+            break;
+        case !!app:
+            items.push({ message: messages.shortApp });
+            break;
+        default:
+            // no-op
+            break;
     }
 
-    if (app) {
-        return { message: messages.shortApp };
+    if (watermark) {
```

Comment:
So far we've only displayed a single bullet for the short version of the summary. Just to confirm: does product/design want to make an exception for watermarking?

Reply:
Yes confirm this is desired with Product.